src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs

```diff
@@ -30,78 +30,33 @@ public StandardService(CacheService cacheService, IAssessmentOrgsApiClient asses
             _standardRepository = standardRepository;
         }
 
-        public async Task<IEnumerable<StandardSummary>> GetAllStandardsV2()
+        public async Task<IEnumerable<StandardCollation>> GetAllStandardsV2()
```

Comment:
Do we need it to say V2 still?

Reply:
Ahh! Good one, was going to rename that. 👍